types/draco3d/index.d.ts

```diff
@@ -147,56 +156,24 @@ export enum GeometryType {}
 // tslint:disable-next-line:no-empty-interface
 export enum DataType {}
 
-export interface Mesh {
+export interface PointCloud {
+    num_attributes: () => number;
+    num_points: () => number;
+}
+
+export interface Mesh extends PointCloud {
```

Comment:
Does this actually extend a PointCloud class in the Draco C++ library? If not I'd prefer not to represent it as a subclass here... mind separating these? I think it's OK to duplicate the two methods above.

Reply:
Yes, from the source `Mesh is just an extended PointCloud with extra connectivity data that defines`
https://github.com/google/draco/blob/master/src/draco/mesh/mesh.h#L52

Reply:
Huh that's interesting! Makes sense to do the same here then thanks. 👍🏻